plv8.cc

```diff
@@ -134,6 +134,16 @@ static plv8_exec_env		   *exec_env_head = NULL;
 extern const unsigned char coffee_script_binary_data[];
 extern const unsigned char livescript_binary_data[];
 
+class Plv8ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
+ public:
+  virtual void* Allocate(size_t length) {
+    void* data = AllocateUninitialized(length);
+    return data == NULL ? data : memset(data, 0, length);
+  }
+  virtual void* AllocateUninitialized(size_t length) { return palloc(length); }
```

Comment:
i can do that if you think it makes sense.  nowhere else in the code is using `PG_TRY` around a `palloc` though.

let me know, and i'm happy to add it.